internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -151,6 +171,7 @@ func (r *ClusterDeploymentsReconciler) updateClusterMetadata(ctx context.Context
 		return err
 	}
 	cluster.Spec.Installed = true
+	cluster.Spec.Platform.AgentBareMetal.APIVIPDNSName = fmt.Sprintf("api.%s.%s", cluster.Spec.ClusterName, cluster.Spec.BaseDomain)
```

Comment:
Isn't it something that should be defined by the user?

Reply:
In this case, the clusterDeployment is not created from scratch but from an existing Cluster that has been installed.
In my opinion it makes sense in this flow to updated it so that the user could add additional nodes without having to update this parameter.
WDYT?

Reply:
Because we have two flows, one is that we convert the cluster and then it make sense to have it here. 
The other flow is that user import the cluster and create the CR, so maybe first you need to check if this value is not already set?

Reply:
I can add a check, though currently this method will not be called if the CR is "imported":
```

	// In case the Cluster is a Day 1 cluster and is installed, update the Metadata and create secrets for credentials
	if *cluster.Status == models.ClusterStatusInstalled && swag.StringValue(cluster.Kind) == models.ClusterKindCluster {
```

Reply:
Added a check.